### DISCUSSION TITLE
"Bad Gateway" links on enterprise site

### DISCUSSION THREAD

**Discussion by CodingAmateur123** [community/community#43558] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I want to sign up for the student developer pack including GitHub Copilot, but everywhere I click, I get a blank "502 Bad Gateway - nginx/1.15.12" screen. I'm gussing this is a issue that needs to be fixed?

**Comment by auchynnikau** [comment]
It's not a client side error, so the problem is not on you side. You have to wait a bit till it will be fixed on the server side. Try to sign up later.

**Comment by otkrickey** [comment]
Solved: #43634
